Registration Form - Step 44

### Describe the Issue

Attempting to submit the following code after refreshing several times generates an error message:

```
fieldset:last-of-type {
    border-bottom: none;
}
```

The browser's console shows this error:

```
dom-test-evaluator.js:2 n {message: "expected '' to be one of [ 'none', 'medium none', 'medium' ]", showDiff: true, actual: '', expected: Array(3), operator: 'deepStrictEqual', …}actual: ""expected: (3) ['none', 'medium none', 'medium']message: "expected '' to be one of [ 'none', 'medium none', 'medium' ]"operator: "deepStrictEqual"showDiff: truestack: "AssertionError: expected '' to be one of [ 'none', 'medium none', 'medium' ]\n    at eval (eval at #a (https://www.freecodecamp.org/js/test-runner/9.0.0/dom-test-evaluator.js:2:255327), <anonymous>:3:8)\n    at eval (eval at #a (https://www.freecodecamp.org/js/test-runner/9.0.0/dom-test-evaluator.js:2:255327), <anonymous>:4:3)\n    at #a (https://www.freecodecamp.org/js/test-runner/9.0.0/dom-test-evaluator.js:2:255327)\n    at DOMTestEvaluator.handleMessage (https://www.freecodecamp.org/js/test-runner/9.0.0/dom-test-evaluator.js:2:255842)\n    at onmessage (https://www.freecodecamp.org/js/test-runner/9.0.0/dom-test-evaluator.js:2:256161)"[[Prototype]]: Error
#a @ dom-test-evaluator.js:2
await in #a
handleMessage @ dom-test-evaluator.js:2
onmessage @ dom-test-evaluator.js:2
```

Forum posts where users report this issue:
https://forum.freecodecamp.org/t/design-a-registration-form-step-44/785930
https://forum.freecodecamp.org/t/design-a-registration-form-step-44/785898

### Affected Page

https://www.freecodecamp.org/learn/responsive-web-design-v9/workshop-registration-form/step-44

### Steps to Reproduce

Try to submit this code:

```
fieldset:last-of-type {
    border-bottom: none;
}
```

### Recommended fix or suggestions

Fix the test?

### Screenshots

_No response_

--- a/curriculum/challenges/english/blocks/workshop-registration-form/62b30924c5e4ef0daba23b5e.md
+++ b/curriculum/challenges/english/blocks/workshop-registration-form/62b30924c5e4ef0daba23b5e.md
@@ -30,8 +30,8 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type'))
 Your `fieldset:last-of-type` should have `border-bottom` set as `none`.
 
 ```js
-const borderBottom = new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type')?.borderBottom;
-assert.oneOf(borderBottom, ['none', 'medium none', 'medium']);
+const borderBottomStyle = new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type')?.borderBottomStyle;
+assert.equal(borderBottomStyle, 'none');
 ```
 
 # --seed--

